docs/transport.rst

```diff
@@ -159,19 +164,19 @@ This workflow could be divided into the following steps:
     1. A publisher application calls the *Client* library to send a given topic.
     2. The *Client* library serializes the topic inside an XRCE message using *Micro CDR*.
        As a result, the XRCE message with the topic is stored in an **Output Stream Buffer**.
-    3. The *Client* library calls the Serial Transport to send the serialized message.
-    4. The Serial Transport frames the message, that is, adds the header, payload, and CRC of the frame, taking into account the stuffing.
+    3. The *Client* library calls the Stream Framing Protocol to send the serialized message.
+    4. The Stream Transport frames the message, that is, adds the header, payload, and CRC of the frame, taking into account the stuffing.
        This step takes place in an auxiliary buffer called **Framing Buffer**.
     5. Each time the Framing Buffer is full, the data is flushed into the **Device Buffer**, calling the writing system function.
 
 .. image:: images/serial_transport_sending.svg
 
 This approach has some advantages which should be pointed out:
 
-    1. The HDLD framing and the CRC control provide **integrity** and **security** to the Serial Transport.
+    1. The HDLD framing and the CRC control provide **integrity** and **security** to the Stream Framing.
     2. The framing technique allows to **reducing memory usage**.
        It is because the Framing Buffer size (42 bytes) bounds the Device Buffer size.
-    3. The framing technique also allows sending **large data** over serial.
+    3. The framing technique also allows sending **large data** over stream oriented transports.
        It is because the message size is not bounded by the Device Buffer size, since the message is fragmented and stuffing during the framing stage.
```

Comment:
```suggestion
       The reason is that the message size is not bounded by the Device Buffer size, since the message is fragmented and has undergone byte stuffing during the framing stage.
```